### PR TITLE
Add Fuzz Introspector to FAQ

### PR DESCRIPTION
**Pull request by Navidem**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -152,6 +152,23 @@ In either case, look at the
...
 for your target(s) and figure out why some parts of the code are not covered. 
 
+## My fuzzer stopped finding new coverage/bugs after a while.
+
+It is common that the fuzzer gets saturated and cannot find new coverage or bugs after a while.
+We have designed and developed [Fuzz Introspector](https://github.com/ossf/fuzz-introspector)
+to give developers a tool to evaluate the fuzzers performance. Fuzz Introspector aims to improve 
+fuzzing experience by guiding developer on determining their fuzzer's bottlenecks. It helps by providing
+aggregated and individual fuzzers reachability and coverage reports. Developer can either introduce a new 
+fuzz target or modify an existing one to improve the quality of the harness. To check some exapmle cases
```
Comment (inferno-chromium) [inline docs/faq.md:162]: example

File: docs/faq.md [hunk 2]
```diff
@@ -152,6 +152,23 @@ In either case, look at the
 [coverage reports]({{ site.baseurl }}/further-reading/clusterfuzz#coverage-reports)
 for your target(s) and figure out why some parts of the code are not covered. 
 
+## My fuzzer stopped finding new coverage/bugs after a while.
```
Comment (inferno-chromium) [inline docs/faq.md:155]: Phrase as a question?

What if my fuzzer does not find new coverage or bugs after a while.

**Review by inferno-chromium — APPROVED** [review]
Also, remember to add stuff to https://google.github.io/oss-fuzz/getting-started/new-project-guide/

File: docs/faq.md [hunk 3]
```diff
@@ -152,6 +152,23 @@ In either case, look at the
 [coverage reports]({{ site.baseurl }}/further-reading/clusterfuzz#coverage-reports)
 for your target(s) and figure out why some parts of the code are not covered. 
 
+## What if my fuzzer does not find new coverage or bugs after a while?
+
+It is common that the fuzzer gets saturated and cannot find new coverage or bugs after a while.
+We have designed and developed [Fuzz Introspector](https://github.com/ossf/fuzz-introspector)
+to give developers a tool to evaluate the fuzzers performance. Fuzz Introspector aims to improve
```
Comment (jonathanmetzman) [inline docs/faq.md:159]: I would replace "to give developers a tool to" with "to help developers"

Comment (jonathanmetzman) [inline docs/faq.md:159]: fuzzer's

File: docs/faq.md [hunk 4]
```diff
@@ -152,6 +152,23 @@ In either case, look at the
 [coverage reports]({{ site.baseurl }}/further-reading/clusterfuzz#coverage-reports)
 for your target(s) and figure out why some parts of the code are not covered. 
 
+## What if my fuzzer does not find new coverage or bugs after a while?
+
+It is common that the fuzzer gets saturated and cannot find new coverage or bugs after a while.
+We have designed and developed [Fuzz Introspector](https://github.com/ossf/fuzz-introspector)
+to give developers a tool to evaluate the fuzzers performance. Fuzz Introspector aims to improve 
+fuzzing experience by guiding developer on determining their fuzzer's bottlenecks. It helps by providing
+aggregated and individual fuzzers reachability and coverage reports. Developer can either introduce a new
```
Comment (jonathanmetzman) [inline docs/faq.md:161]: fuzzer

File: docs/faq.md (hunk 3 above)
Comment (jonathanmetzman) [inline docs/faq.md:159]: maybe "Fuzz introspector helps developers determine their fuzzer's bottlenecks"

Comment (jonathanmetzman) [inline docs/faq.md:159]: the formatting of this seems arbitrary to me. Maybe try to do one sentence per line or an 80-char limit.

File: docs/faq.md [hunk 5]
```diff
@@ -152,6 +152,23 @@ In either case, look at the
...
 for your target(s) and figure out why some parts of the code are not covered. 
 
+## What if my fuzzer does not find new coverage or bugs after a while?
+
+It is common that the fuzzer gets saturated and cannot find new coverage or bugs after a while.
+We have designed and developed [Fuzz Introspector](https://github.com/ossf/fuzz-introspector)
+to give developers a tool to evaluate the fuzzers performance. Fuzz Introspector aims to improve 
+fuzzing experience by guiding developer on determining their fuzzer's bottlenecks. It helps by providing
+aggregated and individual fuzzers reachability and coverage reports. Developer can either introduce a new 
+fuzz target or modify an existing one to improve the quality of the harness. To check some example cases
```
Comment (jonathanmetzman) [inline docs/faq.md:162]: Maybe explain that improving a harness or introducing a new target can cover previously unreached code.

File: docs/faq.md [hunk 6]
```diff
@@ -152,6 +152,23 @@ In either case, look at the
...
 
+## What if my fuzzer does not find new coverage or bugs after a while?
+
+It is common that the fuzzer gets saturated and cannot find new coverage or bugs after a while.
+We have designed and developed [Fuzz Introspector](https://github.com/ossf/fuzz-introspector)
+to give developers a tool to evaluate the fuzzers performance. Fuzz Introspector aims to improve 
+fuzzing experience by guiding developer on determining their fuzzer's bottlenecks. It helps by providing
+aggregated and individual fuzzers reachability and coverage reports. Developer can either introduce a new 
+fuzz target or modify an existing one to improve the quality of the harness. To check some example cases 
+where Fuzz Introspector improved a fuzzing project refere to these
```
Comment (jonathanmetzman) [inline docs/faq.md:163]: Change `To check some example cases where Fuzz Introspector improved a fuzzing project refere to these  [case studies](...)`
to 
`Here are some [case studies](...) where Fuzz Introspector improved fuzzing of a project.`

File: docs/faq.md [hunk 7]
```diff
@@ -152,6 +152,23 @@ In either case, look at the
...
+
+It is common that the fuzzer gets saturated and cannot find new coverage or bugs after a while.
+We have designed and developed [Fuzz Introspector](https://github.com/ossf/fuzz-introspector)
+to give developers a tool to evaluate the fuzzers performance. Fuzz Introspector aims to improve 
+fuzzing experience by guiding developer on determining their fuzzer's bottlenecks. It helps by providing
+aggregated and individual fuzzers reachability and coverage reports. Developer can either introduce a new 
+fuzz target or modify an existing one to improve the quality of the harness. To check some example cases 
+where Fuzz Introspector improved a fuzzing project refere to these 
+[case studies](https://github.com/ossf/fuzz-introspector/blob/main/doc/CaseStudies.md). Fuzz Introspector 
+reports are available on [OSS-Fuzz report page](https://oss-fuzz.com/) or through this
```
Comment (jonathanmetzman) [inline docs/faq.md:165]: "OSS-Fuzz report page" -> "OSS-Fuzz homepage".

File: docs/faq.md [hunk 8]
```diff
@@ -152,6 +152,23 @@ In either case, look at the
...
+to give developers a tool to evaluate the fuzzers performance. Fuzz Introspector aims to improve 
+fuzzing experience by guiding developer on determining their fuzzer's bottlenecks. It helps by providing
+aggregated and individual fuzzers reachability and coverage reports. Developer can either introduce a new 
+fuzz target or modify an existing one to improve the quality of the harness. To check some example cases 
+where Fuzz Introspector improved a fuzzing project refere to these 
+[case studies](https://github.com/ossf/fuzz-introspector/blob/main/doc/CaseStudies.md). Fuzz Introspector 
+reports are available on [OSS-Fuzz report page](https://oss-fuzz.com/) or through this 
+[index](http://oss-fuzz-introspector.storage.googleapis.com/index.html).
+
+Developer can also utilize Fuzz Introspector for their new fuzz target developement by trying it on their
```
Comment (jonathanmetzman) [inline docs/faq.md:168]: Developers

**Pull request merged**